addPlant.php

```diff
@@ -0,0 +1,137 @@
+                is_numeric($inputtedPlantName)) {
+                    echo "Please select use valid characters for the plant name.";
+                }
+                if(intval($inputtedPlantFamily)==0 ||
+                intval($inputtedPlantFamily < 4)) {
+                    echo "Please select a valid plant family.";
+                }
+                if(is_numeric($inputtedScientificName)) {
+                    echo "Please select use valid characters for the scientific name.";
+                }
+                if(strlen($inputtedImage)==0 || 
+                    filter_var($inputtedImage, FILTER_VALIDATE_URL) === FALSE) {
+                    echo "Please enter a valid image URL.";
+                }
+                if(strlen($inputtedDescription)==0 ||
+                strlen($inputtedDescription < 10) ||
+                is_numeric($inputtedDescription)) {
+                    echo "Decscription must be at least 10 characters.";
+                }
+                else {
+
+                $db = new PDO('mysql:host=db; dbname=plants', 'root', 'password');
+                $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+                $plantModel = new PlantModel($db);
+
+                
+                $success=$plantModel->addNewPlant($inputtedPlantName, $inputtedScientificName, $inputtedPlantFamily, $inputtedImage, $inputtedDescription);
+                }
+            }
+            ?>
+
+
+
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Comfortaa:wght@300;400;500;600;700&family=Open+Sans:wght@300;400;500;600;700;800&family=Work+Sans:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="addPlant.css">
+
+        <title>Mr Fancy Plants</title>
+    </head>
+    <body>
+        <header>
+            <div class="logo">
+                <h1> Mr Fancy Plants </h1>
+                <p> Girl, pileas!</p>
+            </div>
+        </header>
+
+        <a class="PLANTBUTTON" href="index.php">Back Home</a>
+        
+        <form method="POST" class="add_plant">
+
+                <label for="newPlant">Plant Name</label>
+                <input type="text" name="newPlantName" />
+
+                <label for="newPlantFamily">Select the family</label>
+                <select name="newPlantFamily">
+                    <option value="0">--Plant Family--</option>
+                    <option value="4">Foliage Plants</option>
+                    <option value="1">Succlents and Cacti</option>
+                    <option value="3">Flowing Plants</option>
+                    <option value="2">Trailing or Hanging Plants</option>
```

Comment:
You should now work on getting these out of the database, idea being that if you ended up building an add plant family page, you wouldn't have to update this dropdown list every time a user adds a family.